README.md

```diff
@@ -10,7 +10,7 @@ The project uses [repo deploy keys](https://developer.github.com/v3/guides/manag
 
 ### Update versions
 
-The project can update the version of the documentation using the `package.json` version as its reference.
+The project can update the version of the documentation using the `package.json` semantic version as its reference. This will only work with minors or patch updates for the first documentation versioning or in case of a major update you will need to perform this update manually. [Here](https://docusaurus.io/docs/en/versioning) you can check the steps to have the manual version update.
```

Comment:
```suggestion
The project can update the version of the documentation using the `package.json` semantic version as its reference. This will only work with minors or patch updates for the first documentation versioning or in case of a major update you will need to perform this update manually. [Here](https://docusaurus.io/docs/en/versioning) you can check the steps to do the manual version update.
```

Reply:
Nice catch!!!